### PR TITLE
Add Setting for toolkit requirement for wheel repair

### PR DESCRIPTION
**Pull request by PabstMirror**
Add ace setting to allow replace/remove wheels without a toolkit.

### CODE REVIEW

**Comment by alganthe** [comment]
:+1:

**Comment by Glowbal** [comment]
Very nice. I like it :+1:

File: addons/repair/stringtable.xml [hunk 1]
```diff
@@ -901,5 +901,11 @@
             <Polish>Ilość wybranych części zamiennych.</Polish>
             <Portuguese>Número de partes sobressalentes.</Portuguese>
         </Key>
+        <Key ID="STR_ACE_Repair_wheelRepairRequiredItems_name">
+            <English>Req Items for wheel repair</English>
```
Comment (SilentSpike) [inline addons/repair/stringtable.xml]: I'd maybe make this something like "Wheel repair requirements", cleaner and actually 1 character less if space is an issue

**Comment by commy2** [comment]
:+1:

**Pull request merged**